### PR TITLE
[10.0][MIG] connector_job_subscribe

### PR DESCRIPTION
**Pull request by sbidoul**

### CODE REVIEW

**Comment by sbidoul** [comment]
@Cedric-Pigeon @lmignon

File: queue_job_subscribe/views/res_users_view.xml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<openerp>
```
Comment (lmignon) [inline queue_job_subscribe/views/res_users_view.xml]: @sbidoul this file should be migrated to use the new conventions.

**Review by lmignon — CHANGES_REQUESTED** [review]

**Comment by sbidoul** [comment]
@lmignon done

**Review by lmignon — APPROVED** [review]
@sbidoul Thank you for the migration

**Review by Cedric-Pigeon — APPROVED** [review]
Code review and tests

File: queue_job_subscribe/README.rst [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/queue_job/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback
+`here <https://github.com/OCA/queue_job/issues/new?body
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
```
Comment (simahawk) [inline queue_job_subscribe/README.rst]: this link sounds broken

**Review by simahawk — APPROVED** [review]
LGTM. Just a small issue with a link in the README.

**Comment by sbidoul** [comment]
@simahawk I fixed that and improved the readme. Thanks for reviewing!

**Comment by simahawk** [comment]
\o/

**Pull request merged**